resources/public/api/conf/1.0/schemas/retrieve_coding_out_underpayments_response.json

```diff
@@ -0,0 +1,112 @@
+      "type": "array",
+      "description": "Array containing details of coding out PAYE underpayments.",
+      "items": {
+        "amount": {
+          "description": "The total coding out amount attributed to PAYE underpayments. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "type": "number",
+          "multipleOf": 0.01,
+          "minimum": 0.00,
+          "maximum": 99999999999.99,
+          "example": "5000.99"
+        },
+        "relatedTaxYear": {
+          "description": "The tax year related to the underpayment.",
+          "type": "string",
+          "example": "2021-22"
+        },
+        "submittedOn": {
+          "description": "The date the coding out information was added. It is either the latest change date held within HMRC or the latest user submitted date. It is in the format YYYY-MM-DDThh:mm:ssZ.",
+          "type": "string",
+          "example": "2020-07-06T09:37:17Z"
+        }
+      }
+    },
+    "debts": {
+      "type": "array",
+      "description": "Array containing details of coding out debts.",
+      "items": {
+        "amount": {
+          "description": "The amount of the debt. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "type": "number",
+          "multipleOf": 0.01,
+          "minimum": 0.00,
+          "maximum": 99999999999.99,
+          "example": "5000.99"
+        },
+        "relatedTaxYear": {
+          "description": "The tax year related to the debt.",
+          "type": "string",
+          "example": "2021-22"
+        },
+        "submittedOn": {
+          "description": "The date the coding out information was added. It is either the latest change date held within HMRC or the latest user submitted date. It is in the format YYYY-MM-DDThh:mm:ssZ.",
+          "type": "string",
+          "example": "2020-07-06T09:37:17Z"
+        }
+      }
+    },
+    "inYearAdjustments": {
+      "type": "object",
+      "description": "Object containing details of in-year adjusted coding out underpayments and debts.",
+      "items": {
+        "amount": {
+          "description": "The amount of the in-year adjustment. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "type": "number",
+          "multipleOf": 0.01,
+          "minimum": 0.00,
+          "maximum": 99999999999.99,
+          "example": "5000.99"
+        },
+        "relatedTaxYear": {
+          "description": "The tax year related to the in-year adjustment.",
+          "type": "string",
+          "example": "2021-22"
+        },
+        "submittedOn": {
+          "description": "The date the coding out information was added. It is either the latest change date held within HMRC or the latest user submitted date. It is in the format YYYY-MM-DDThh:mm:ssZ.",
+          "type": "string",
+          "example": "2020-07-06T09:37:17Z"
+            }
+          }
+        }
+      }
+    }
```

Comment:
HATEOAS links also need to be included here, like: https://github.com/hmrc/self-assessment-accounts-api/blob/a52324a272af32544fe7fa9da3b635bf5153c8f1/resources/public/api/conf/1.0/schemas/create_or_amend_coding_out_underpayments_response.json#L7